### DISCUSSION TITLE
How o add a field like Address or Place (without using extra)?

### DISCUSSION THREAD

**Discussion by thistlillo** [retorquere/zotero-better-bibtex#2207] [General]
Many items in my library are papers in conferences. I know I can use the `extra` field, but I would like a `place`/`address` field for every entry in my library.

Is there any way to add a field like `address` or `place` in the default set that is shown inside of Zotero (and that are immediately editable)?

**Comment by retorquere** [comment]
The Zotero conference paper item type already has a Place field.

Technically it would be possible to add more "regular" fields from a plugin, but it is hard to do robustly, and I don't think it fits within the scope of BBT to add general fields.

**Comment by thistlillo** [comment]
Thank you. I will just switch item type then (the op is longer than filling an available field, that was the reason of my question). Again, thanks.
